docs/en/wallet-instance-attestation.rst

```diff
@@ -118,7 +119,7 @@ Payload
 ||       ||                                                            |
 +--------+-------------------------------------------------------------+
 || nonce || The nonce obtained from the                                |
-||       || Wallet Provider.                                           |
+||       || App Attestation Service.                                   |
```

Comment:
Ok, can I ask you to add the App Attestation Service here
https://github.com/italia/eudi-wallet-it-docs/blob/versione-corrente/docs/en/defined-terms.rst#defined-terms

Reply:
Already added https://github.com/italia/eudi-wallet-it-docs/pull/121/files#diff-0b13b354d3cc69c57166e38bca0c889a5e024419acaed05995f6c1ffe5d3b6e4R40